.travis.yml

```diff
@@ -39,16 +39,16 @@ matrix:
       language: c
       env: NOTMUCHVER=0.28 GHCHEAD=true
       addons: {apt: {packages: [elinks,tmux,g++,libxapian-dev,libgmime-2.6-dev,libtalloc-dev,zlib1g-dev,ghc-ppa-tools,cabal-install-head,ghc-head], sources: [hvr-ghc]}}
-    - compiler: "ghc863 (default)"
+    - compiler: "ghc865 (default)"
       language: nix
       before_install:
       install:
       script:
        - nix-channel --add https://nixos.org/channels/nixos-19.09 nixpkgs
        - nix-channel --update
        - nix-env --install --attr nixpkgs.elinks
-       - nix-build
-       - nix-shell --command 'PATH=$PATH:result/bin cabal test --show-details=streaming uat'
+       - nix-build -j 2
```

Comment:
what about `-j auto` (according to https://www.mankier.com/1/nix-build)?

Reply:
Yeah actually this sounds like a sensible suggestion. Will change it :)